src/resources/postcss/views/skeleton/_single-event.pcss

```diff
@@ -260,3 +284,83 @@
 		}
 	}
 }
+
+/* Related Events block */
+.tribe-events-related-events-title,
+.tribe-related-events {
+	order: 2;
+}
+
+.tribe-events-related-events-title {
+	margin-bottom: var(--spacer-5);
+	padding-top: var(--spacer-7);
+	width: 100%;
+}
+
+.tribe-events-single ul.tribe-related-events {
+	display: flex;
+	flex-direction: column;
+	margin-bottom: 0;
+	
+	@media (--viewport-medium) {
+		flex-direction: row;
+		justify-content: space-between;
+	}
+	
+	li {
+		display: flex;
+		flex: 1;
+		flex-direction: column;
+		margin-bottom: var(--spacer-7);
+		padding: 0;
+		width: auto;
+		
+		&:last-child {
+			margin-right: 0;
+		}
+		
+		.tribe-related-events-thumbnail {
+			margin-bottom: var(--spacer-3);
+			width: 100%;
+			
+			/* Display featured images in a specific aspect ratio */
+			@media (--viewport-medium) {
+				a {
+					display: block;
+					overflow: hidden;
+					position: relative;
+					padding-top: 75%; /* 4:3 aspect ratio */
+				}
+				
+				img {
+					height: 100%;
+					object-fit: cover;
```

Comment:
this is not supported in IE11, we should probably find another way to do this.

Reply:
will remove the logic for now and revisit in a different PR, Rachel said it's more of a nice to have at this point